docs/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [Unreleased]
+
+### Added
+
+- New `EVM.NodePool.SelectionMode` `TotalDifficulty` to use the node with the greatest total difficulty.
```

Comment:
I'm not sure where this is supposed to be since the unreleased set is for 1.10

Reply:
Why tangle it back up on purpose? We have accidentally included/omitted things in releases before because of this confusing practice, and recently have revised the release process specifically to address this very problem.

cc @HenryNguyen5 What is the proper approach here?

Reply:
I talked to @chainchad about this and we concluded that we can just leave it tangled for now, we havent officially proposed a process for a better changelog practice.